In-house patch to remove unnecessary dependencies from Neutron's
requirements files. The specific reasons are as follows:

oslo.rootwrap	Not applicable to Solaris

paste		Not applicable

--- neutron-2014.2.2/neutron.egg-info/requires.txt.~1~	2015-02-05 07:46:25.000000000 -0800
+++ neutron-2014.2.2/neutron.egg-info/requires.txt	2015-02-25 00:20:01.378154156 -0800
@@ -1,5 +1,4 @@
 pbr>=0.6,!=0.7,<1.0
-Paste
 PasteDeploy>=1.5.0
 Routes>=1.12.3,!=2.0
 anyjson>=0.3.3
@@ -24,5 +23,4 @@ stevedore>=1.0.0  # Apache-2.0
 oslo.config>=1.4.0  # Apache-2.0
 oslo.db>=1.0.0,<1.1  # Apache-2.0
 oslo.messaging>=1.4.0,!=1.5.0,<1.6.0
-oslo.rootwrap>=1.3.0
-python-novaclient>=2.18.0
+python-novaclient>=2.18.0
\ No newline at end of file
--- neutron-2014.2.2/requirements.txt.~1~	2015-02-05 07:45:33.000000000 -0800
+++ neutron-2014.2.2/requirements.txt	2015-02-25 00:19:55.500774949 -0800
@@ -3,7 +3,6 @@
 # process, which may cause wedges in the gate later.
 pbr>=0.6,!=0.7,<1.0
 
-Paste
 PasteDeploy>=1.5.0
 Routes>=1.12.3,!=2.0
 anyjson>=0.3.3
@@ -28,6 +27,5 @@ stevedore>=1.0.0  # Apache-2.0
 oslo.config>=1.4.0  # Apache-2.0
 oslo.db>=1.0.0,<1.1  # Apache-2.0
 oslo.messaging>=1.4.0,!=1.5.0,<1.6.0
-oslo.rootwrap>=1.3.0
 
 python-novaclient>=2.18.0
